docs/lib/pubsub/fragments/js/getting-started.md

```diff
@@ -82,3 +82,30 @@ Amplify.addPluggable(new MqttOverWSProvider({
 ```
 
 You can integrate any MQTT Over WebSocket provider with your app. Click [here](https://docs.aws.amazon.com/iot/latest/developerguide/protocols.html#mqtt-ws) to learn more about MQTT Over WebSocket.
+
+## Removing a Provider
```

Comment:
```suggestion
## How to reconfigure PubSub provider during runtime
```

While this is technically accurate, the customer use case is likely as highlighted below in your text.

Reply:
Thanks, @renebrandel! Definitely agree with you on this.

Reply:
nit: Perhaps we can update this to: `How to reconfigure PubSub providers during runtime`?